src/components/status.js

```diff
@@ -0,0 +1,240 @@
+  }
+
+  tr:first-child th:last-child,
+  td:last-child {
+    border-right-width: 1px;
+  }
+
+  tr:last-child td,
+  tbody tr:last-child th {
+    border-bottom-width: 1px;
+  }
+
+  td {
+    vertical-align: middle;
+  }
+`
+
+const STATUS_COLORS = {
+  alpha: 'severe.emphasis',
+  beta: 'attention.emphasis',
+  stable: 'success.emphasis',
+  deprecated: 'danger.emphasis'
+}
+
+function getStatusColor(status) {
+  return STATUS_COLORS[status.toLowerCase()] || 'fg.muted'
+}
+
+const initialFieldTypes = [
+  {type: '', name: 'All components', icon: ListUnorderedIcon},
+  {type: 'Accessibility', name: 'Reviewed for accessibility', icon: AccessibilityInsetIcon}
+]
+
+const statusFieldTypes = [
+  {type: 'Stable', name: 'Stable'},
+  {type: 'Beta', name: 'Beta'},
+  {type: 'Alpha', name: 'Alpha'},
+  {type: 'Draft', name: 'Draft'},
+  {type: 'Deprecated', name: 'Deprecated'}
+]
+
+export function StatusTable() {
+  const [components, setComponents] = React.useState([])
+  const [selectedField, setSelectedField] = React.useState(initialFieldTypes[0])
+
+  React.useEffect(() => {
+    getComponents()
+      .then(components => setComponents(components))
+      .catch(error => console.error(error))
+  }, [])
+
+  return (
+    <ThemeProvider colorMode="day" nightScheme="dark_dimmed">
+      <Box sx={{paddingBottom: 4}}>
+        <ActionMenu>
+          <ActionMenu.Button aria-label="Show components">Show: {selectedField.name}</ActionMenu.Button>
+          <ActionMenu.Overlay width="medium">
+            <ActionList selectionVariant="single">
+              <ActionList.Group>
+                {initialFieldTypes.map((field, index) => (
+                  <ActionList.Item
+                    key={index}
+                    selected={field.type === selectedField.type}
+                    onSelect={() => setSelectedField(field)}
+                  >
+                    <ActionList.LeadingVisual>
+                      <StyledOcticon icon={field.icon} />
+                    </ActionList.LeadingVisual>
+                    {field.name}
+                  </ActionList.Item>
+                ))}
+              </ActionList.Group>
+              <ActionList.Divider />
+              <ActionList.Group title="Status">
+                {statusFieldTypes.map((field, index) => (
+                  <ActionList.Item
+                    key={index}
+                    selected={field.type === selectedField.type}
+                    onSelect={() => setSelectedField(field)}
+                  >
+                    <ActionList.LeadingVisual>
+                      <StyledOcticon icon={DotFillIcon} color={getStatusColor(field.type)} />
+                    </ActionList.LeadingVisual>
+                    {field.name}
+                  </ActionList.Item>
+                ))}
+              </ActionList.Group>
+            </ActionList>
+          </ActionMenu.Overlay>
+        </ActionMenu>
+      </Box>
+      {components ? (
+        <Table>
```

Comment:
The table currently overflows the page:

https://user-images.githubusercontent.com/4608155/215540594-4e3f78e2-b37e-470e-916e-a54cc05a7500.mp4

A quick solution to this might be to wrap the whole `<Table>` in a `<Box sx={{overflow: 'auto'}}>`:

Reply:
Yes, I realised this and also I wasn't sure if we will going to display the nav on the left hand side as we don't on primer.style/status. I haven't done any position alignment really for the table. I wrote the things I wasn't clear on what we are going to do on the description.

>On the current [primer.style/status](https://primer.style/status) we don't have the main navigation on the left hand. Do we want the same for the [primer.style/design/status](https://primer-23eb43332d-26441320.drafts.github.io/status) as well? Because currently the table is not aligned as there is not enough space in the main content area to align it when considered the paddings around. cc @emilybrick